dc/council/code/titles/1/sections/1-204.63.xml

```diff
@@ -5,12 +5,12 @@
   <para>
     <num>(a)</num>
     <text>After each act of the Council of the District of Columbia under § <cite path="§1-204.62">1-204.62</cite>(a) authorizing the issuance of general obligation bonds has taken effect, the Mayor shall publish such act at least once in at least 1 newspaper of general circulation within the District together with a notice that such act has taken effect. Each such notice shall be in substantially the following form:</text>
-    <para>
-      <heading>NOTICE</heading>
-      <text>The following act of the Council of the District of Columbia (published with this notice) authorizing the issuance of general obligation bonds has taken effect. As provided in the District of Columbia Home Rule Act, the time within which a suit, action, or proceeding questioning the validity of such bonds may be commenced expires at the end of the 20-day period beginning on the date of the first publication of this notice.</text>
-      <text>. . . . . . . . . . . . .</text>
-      <text>Mayor</text>
-    </para>
+    <text>
+    <strong>NOTICE</strong>
+    <br/>
```

Comment:
> What is the purpose this `<br/>`?

I think either is appropriate. The `<br/>` is a line-break. Since I am changing this file, I will make it a `<text>` element.

> NOTICE is also centered (according to the word document Division I.docx).

Will get it centered.